### PR TITLE
Fix for issue #23

### PR DESCRIPTION
**Pull request by rs028**
Add pressure dependence to RH-H2O conversion and correct the equations. Also fixes a mistake in calcM (to be renamed calcAirDensity after PR #194). It also removes calcPressure, as it will no longer be needed after PR #194.

### CODE REVIEW

**Comment by rs028** [comment]
Note: only the style test fails.

File: conversionFunctions.f90 [hunk 1]
```diff
@@ -31,42 +31,26 @@ pure function calcM( press, temp ) result ( m )
...
+    ! pressure in mbar (1 mbar = 1 hPa), temperature in K
     implicit none
 
     real(kind=DP), intent(in) :: rh, temp, press
-    real(kind=DP) :: h2o, exponent, e1
+    real(kind=DP) :: h2o, h2o_ppm, temp_c, wvp
 
-    exponent = exp( -1.00d00 * ( 597.30d00 - 0.57d00 * ( temp - 273.16d00 ) ) * &
-                   18.00d00 / 1.986d00 * ( 1.00d00 / temp - 1.00d00 / 273.16d00 ) )
+    temp_c = temp - 273.15
```
Comment (spco) [inline conversionFunctions.f90]: Could you write a comment detailing where this calculation comes from, both the document and the equations? As far as I can tell, it looks like the combination of (6) and (1), then (18) in the Vaisala document. The conversion from h2o_ppm to h2o I also understand, but is there an explicit formula we can link to, or at least write in the comment?

File: conversionFunctions.f90 [hunk 2]
```diff
@@ -31,42 +31,26 @@ pure function calcM( press, temp ) result ( m )
     real(kind=DP), intent(in) :: press, temp
     real(kind=DP) :: m, press_pa
 
-    press_pa = press * 1.0d+2
-    m = ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
+    press_pa = press * 1.0d+02
+    m = 1.0d-06 * ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
```
Comment (spco) [inline conversionFunctions.f90]: You previously said that this calculation was correct: https://github.com/AtChem/AtChem/issues/133#issuecomment-299840245 . Has this now changed, or is there confusion over the multiplications by 1e6 and 1e-6 at lines 51-2?

Comment (rs028) [inline conversionFunctions.f90]: Okay, now I am getting confused. I will double check later today or tomorrow.

**Comment by rs028** [comment]
If this passes the tests, lets merge it. I will address the issue with the equation for M in PR #194.

**Comment by rs028** [comment]
Ok, good to go.

**Comment by rs028** [comment]
Yes, the multiplication for 1e-6 should be in there. I think what happened was an error in copy&paste, because I tested the function externally. PR #194 is ready anyways, just need to fix the test results.

**Pull request merged**